### ISSUE TITLE
Login 400: Bad Request

### ISSUE THREAD

**Issue by jmbott** [SEL-Columbia/dokomoforms#248]
https://uganda.dokomoforms.org/

If attempting to login without typing an email you get a 400:Bad request page. 

![image](https://cloud.githubusercontent.com/assets/7769317/22114273/dcb05340-de36-11e6-96d0-b57355860f2c.png)
_______________________________________

It should probably do something like the following, 

![image](https://cloud.githubusercontent.com/assets/7769317/22114340/20e522fc-de37-11e6-9720-26284da76f92.png)

**Comment by vr2262** [comment]
Yeah, the default input box behavior is browser-dependent. "Polished" solution is in the works...
